web/templates/web/header.html

```diff
@@ -6,78 +6,92 @@
 
 <link rel="stylesheet" type="text/css" href="{% static "web/css/header.css" %}">
 
-<div class="nav">
-    <div class="burger">
-        <i class="sidebar icon"></i>
-    </div>
+<header id="header">
+    <div class="logo-centering">{# Empty div for centering logo (on mobile), specified with grid-template-columns #}</div>
+
     <div class="logo">
+        <div>{# Empty div for spacing left of logo, specified with grid-template-columns #}</div>
         <a href="{% url 'front-page' %}">
             <img class="limg large" src="{% static "web/img/logo_white.svg" %}">
             <img class="limg medium" src="{% static "web/img/logo_make.svg" %}">
             <img class="limg small" src="{% static "web/img/logo_m.svg" %}">
         </a>
     </div>
-    <div class="ui secondary inverted menu">
-        <a class="item" href="{% url 'front-page' %}">
-            {% trans "Home" %}
+
+    <div id="burger">
+        <i class="sidebar icon"></i>
+    </div>
+
+    <nav id="nav" class="ui secondary inverted menu">
+        <a class="nav-item item" href="{% url 'events' %}">
```

Comment:
I do not really see the need for the class `nav-item`. I personally find using the selector `#nav .item` to be just as descriptive and much easier to read than `#nav .nav-item`.

Reply:
Mostly because there are `.item`s within `#nav` that should not be selected, and because the `.nav-item`s are very specific "items", in contrast to the ubiquitous `.item`. Can fix it with `#nav > .item`, though, or a change of name, like `.nav-button`. What do you think?

Reply:
I think `#nav > .item` is good. I tells me that all items in the upper level of the navigation bar should be selectable (etc.).